### DISCUSSION TITLE
Jellyfin SSO and Twitch

### DISCUSSION THREAD

**Discussion by DerGardine** [9p4/jellyfin-plugin-sso#94] [Q&A]
Hey,
I managed to make the plugin work with Twitch with some adjustements and now I'm wondering how to check the response.
I added the scope to access subscription status with user:read:subscriptions and get correctly asked for the right to view it.

How do I now check for tier:2000 in the answer?

This is an example answer:

{
  "data": [
    {
      "broadcaster_id": "141981764",
      "broadcaster_login": "twitchdev",
      "broadcaster_name": "TwitchDev",
      "gifter_id": "12826",
      "gifter_login": "twitch",
      "gifter_name": "Twitch",
      "is_gift": true,
      "tier": "1000",
      "plan_name": "Channel Subscription (twitchdev)",
      "user_id": "527115020",
      "user_name": "twitchgaming",
      "user_login": "twitchgaming"
    },
    ...
  ],
  "pagination": {
    "cursor": "xxxx"
  },
  "total": 13,
  "points": 13
}

And I want to access the tier value and check if it is 2000.
Any help is appreciated!

Julian

**Comment by 9p4** [comment]
Does that JSON come back in the OpenID response?
